### ISSUE TITLE
Information about: spa_navigation

### ISSUE THREAD

**Issue by sukratkashyap** [akamai/boomerang#280]
My question is how many times will `spa_navigation` event will be fired in a SPA app. What I am seeing right now is that `spa_navigation` is only called once at the `spa_hard` navigation and then for each `spa` navigations I am only seeing `spa_init` event being fired?

**Comment by nicjansma** [comment]
@sukratkashyap you're right, it looks like `spa_navigation` is only being triggered if an `onComplete` function is used during `route_change()`, and during all Hard navigations.  We can make sure to fire `spa_navigation` in all cases.

https://github.com/akamai/boomerang/blob/master/plugins/spa.js#L434

The other way would be to detect `beacon` or `before_beacon` and look at `http.initiator=spa`.
